spec/models/case_contact_spec.rb

```diff
@@ -2,8 +2,8 @@
 
 RSpec.describe CaseContact, type: :model do
   context "validations" do
-    it { should validate_numericality_of(:miles_driven).is_less_than 10000 }
-    it { should validate_numericality_of(:miles_driven).is_greater_than_or_equal_to 0 }
+    it { is_expected.to validate_numericality_of(:miles_driven).is_less_than 10_000 }
+    it { is_expected.to validate_numericality_of(:miles_driven).is_greater_than_or_equal_to 0 }
```

Comment:
Why this change? Not saying it's wrong, just curious.

Reply:
this is the new rspec style 
I have used https://www.betterspecs.org/#expect in the past and I believe it is one of the influences for this change.